numpy/_core/src/npysort/aquicksort.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef NUMPY_SRC_COMMON_NPYSORT_AQUICKSORT_HPP
+#define NUMPY_SRC_COMMON_NPYSORT_AQUICKSORT_HPP
+
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+#include "npy_cpu_features.h"
```

Comment:
Not sure if we need all these headers. Could you confirm if we really need them?